Makefile

```diff
@@ -54,7 +54,10 @@ src:
 	@$(MAKE) --no-print-directory -C src
 
 test:
-	@$(MAKE) --no-print-directory -C test test
+	@CC=occlum-gcc CXX=occlum-g++ $(MAKE) --no-print-directory -C test test
+
+test-glibc:
+	@CC=gcc CXX=g++ $(MAKE) --no-print-directory -C test test
```

Comment:
```
$ cd <occlum_project> && make test
```
should be equivalent to
```
$ cd <occlum_project>/test && make test
```
That is, the root Makefile simply redirects commands to sub Makefiles. So the Glibc support should apply equally well to both the root Makefile and the sub Makefile in `test/`.

One way to achieve this is to add an env var named `OCCLUM_TEST_GLIBC`. Use this env var to decide the values of `CC` and `CXX` in `test/Makefile`.